README.md

```diff
@@ -51,12 +51,12 @@ generated with markdown-toc</a></i></small>
 
 ## Getting Started 
 
-Please refer to our [Quick Start Guide](https://www.conjur.org/get-started/quick-start/oss-environment/) for detailed information on using Conjur OSS for the first time, or, refer to the 
+Please refer to our [Quick Start Guide](https://www.conjur.org/get-started/quick-start/oss-environment/) for detailed information on using Conjur Open Source for the first time, or, refer to the 
```

Comment:
The first time we are mentioning conjur we should use the long term:
CyberArk Conjur Secrets Manager Open Source